README.md

```diff
@@ -22,22 +27,30 @@ with pm.Model():
 
 ### What belongs in `pymc-experimental`?
 
-- newly-implemented statistical methodologies
+- newly-implemented statistical methods, for example diagnostics or step methods
```

Comment:
Diagnostics belongs to arviz unless there is a strong reason against it.

Reply:
```suggestion
- newly-implemented statistical methods, for example step methods or model construction helpers
```